### PR TITLE
refactor: responsive breakpoints now work with max-width and 3 entries

### PR DESCRIPTION
**Pull request by dkonieczek**

### CODE REVIEW

File: packages/snap-preact/src/Templates/Stores/ThemeStore.test.ts [hunk 1]
```diff
@@ -68,6 +68,7 @@ const testTheme: Theme = {
 			components: {
 				results: {
 					columns: 1,
+					treePath: 'asdf',
```
Comment (korgon) [inline packages/snap-preact/src/Templates/Stores/ThemeStore.test.ts:71]: This necessary?

File: packages/snap-preact/src/Templates/Stores/ThemeStore.tsx
Comment (korgon) [inline packages/snap-preact/src/Templates/Stores/ThemeStore.tsx:1]: Good changes.

**Review by korgon — COMMENTED** [review]
Nice!

File: packages/snap-preact/src/Templates/Stores/ThemeStore.test.ts (hunk 1 above)
Comment (dkonieczek) [reply packages/snap-preact/src/Templates/Stores/ThemeStore.test.ts]: whoops. Cleaned this up

**Pull request merged**